src/oaklib/cli.py

```diff
@@ -1072,6 +1079,13 @@ def annotate(
     impl = settings.impl
     writer = _get_writer(output_type, impl, StreamingYamlWriter, datamodels.text_annotator)
     writer.output = output
+    terms_to_remove = []
```

Comment:
consider putting this into a utility function, to avoid repeating the same chunk of code

Reply:
Done.